lib/core/src/Cardano/Pool/DB.hs

```diff
@@ -191,6 +192,10 @@ data DBLayer m = forall stm. (MonadFail stm, MonadIO stm) => DBLayer
         -> stm ()
         -- ^ Store metadata fetched from a remote server.
 
+    , delPoolMetadata
+        :: stm ()
+        -- ^ Delete all pool metadata.
```

Comment:
I think @rvl's already suggested to rename this from `delPool` to `removePoolMetadata` or `prunePoolMetadata` for consistency with the rest of the interface. I second this :+1: